lucene/highlighter/src/java/org/apache/lucene/search/uhighlight/Passage.java

```diff
@@ -40,7 +40,7 @@
     BytesRef matchTerms[] = new BytesRef[8];
     int numMatches = 0;
 
-    void addMatch(int startOffset, int endOffset, BytesRef term) {
+    public void addMatch(int startOffset, int endOffset, BytesRef term) {
```

Comment:
Excellent; now it's possible for someone to override `FieldHighlighter.highlightOffsetsEnums()` to make Passages.  But do add @lucene.experimental.